templates/boltzmann/middleware/cors.ts

```diff
@@ -32,6 +33,21 @@ function handleCORS ({
         )
       )
 
+      void `{% if honeycomb %}`
+
+      const spanAttributes = {
+        'boltzmann.http.origin': String(context.headers.origin)
```

Comment:
It strikes me that we might do some follow-on work after this PR to document all of the span attributes boltzmann produces and when it produces them.

Reply:
It might be worth doing here and now depending on what you have in mind. We can zoom about this too, and if nothing else do up a ticket.

Reply:
I went ahead and filed issue #109 to capture this as future work.